static/test/javascripts/spec/commercial/hosted/next-video-autoplay.spec.js

```diff
@@ -78,13 +85,8 @@ define([
         });
 
         it('should not trigger autoplay when there is no next video', function (done) {
-            fixturesConfig = {
-                id: 'next-video-autoplay',
-                fixtures: [
-                    '<div class="js-autoplay-timer" data-next-page="">10s</div>'
-                ]
-            };
-            $fixturesContainer = fixtures.render(fixturesConfig);
+            fixtures.clean(fixturesConfig.id);
```

Comment:
I need to remove the html used in the other tests (which includes '<video>' tag) otherwise the test fails! Is it cleaner to put this into a separate `describe`? (I tried this but had to duplicate most of the set up)